### PR TITLE
Adds tweeezers and changes known_implants to a global var

### PR DESCRIPTION
**Pull request by TiviPlus**
## About The Pull Request

The ONLY way to get them is MO and medic

Decreased medical skill makes you deal much more dmage and take longer to pull out shrapnel

## Why It's Good For The Game

FF is bad and this makes it more manageable without removing the shrapnel mechanic

## Changelog
:cl:
add: Adds medic tweezers for removing shrapenel
spriteadd: Sprite by Daytona
code: known_implants is now a managed global
/:cl:

### CODE REVIEW

**Comment by Losenis** [comment]
They are gigantic in terms of space, comparable to a full stack of metal sheets. Also, any plans to make this also come inside the surgical trays orderable in req, along with increasing the surgery vest space by 1 to fit them?

**Comment by Pariah919** [comment]
these should be small items probably

**Comment by TiviPlus** [comment]
Yes

File: code/datums/elements/shrapnel_removal.dm [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+	user.visible_message("<span class='notice'>[user] starts searching for shrapnel in [target] with the [removaltool].</span>", "<span class='notice'>You start searching for shrapnel in [target] with the [removaltool].</span>")
+	if(!do_after(user, do_after_time, TRUE, target, BUSY_ICON_MEDICAL))
+		return
+	for(var/obj/item/I as() in targetlimb.implants)
+		if(is_type_in_list(I, GLOB.known_implants))
+			continue
+		I.unembed_ourself(FALSE)
+		if(skill < SKILL_MEDICAL_PRACTICED)
+			user.visible_message("<span class='notice'>[user] violently rips out [I] from [target]!</span>", "<span class='notice'>You violently rip out [I] from [target]!</span>")
+			target.apply_damage(30 * (SKILL_MEDICAL_PRACTICED - skill), def_zone = user.zone_selected)
```
Comment (XSlayer300) [inline code/datums/elements/shrapnel_removal.dm:46]: Make them use the scream emote.

File: code/datums/elements/shrapnel_removal.dm [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+		else
+			user.visible_message("<span class='notice'>[user] pulls out [I] from [target]!</span>", "<span class='notice'>You pull out [I] from [target]!</span>")
+			target.apply_damage(15, def_zone = user.zone_selected)
```
Comment (XSlayer300) [inline code/datums/elements/shrapnel_removal.dm:49]: And if you want, make them use the pain emote.

File: code/game/objects/items/devices/tweezers.dm [hunk 3]
```diff
@@ -0,0 +1,10 @@
+/obj/item/tweezers
+	name = "medical tweezers"
+	desc = "Medical tweezers intended to remove shrapnel from patients."
+	icon_state = "tweezers"
+	item_state = "tweezers"
+	flags_item = NOBLUDGEON
```
Comment (Losenis) [inline code/game/objects/items/devices/tweezers.dm:6]:
```suggestion
/obj/item/tweezers
	name = "medical tweezers"
	desc = "Medical tweezers intended to remove shrapnel from patients."
	w_class = WEIGHT_CLASS_TINY
	icon_state = "tweezers"
	item_state = "tweezers"
	flags_item = NOBLUDGEON
```

missing w_class

**Review by SplinterGP — APPROVED** [review]

**Pull request merged**